### DISCUSSION TITLE
Button/Link to clear all filters

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#4194] [Q&A] [answered]
Button/Link to clear all filters feature request.

**Comment by crynobone** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/172966/167319204-f64476b0-ef25-4d96-b832-73f64a06abe8.png)

**Reply by ziming** [reply]
I see, it doesn't look like a button hence I missed it. Apologies
